bin/phpstan/constants.php

```diff
@@ -5,6 +5,9 @@
  * @package performance-lab
  */
 
+define( 'TESTS_PLUGIN_DIR', './' );
+define( 'WPINC', 'wp-includes' );
```

Comment:
I guess `WPINC` should be added to https://github.com/szepeviktor/phpstan-wordpress

Reply:
I know there is no core function to get the value of WPINC but I do not want to encourage the users of this package to use WPINC.